src/legacy_util.cpp

```diff
@@ -62,18 +62,24 @@
 #  include <tchar.h>
 #endif
 
+// TODO: remaining non-const globals
+// move Config::log_file() into a global?
+// would enable init_log() without args again
+
```

Comment:
All the log / syslog related state should probably stay global.

Reply:
Yes, leave them for now.